### PR TITLE
Update JS & Add styles

### PR DESCRIPTION
**Pull request by jordszbordsz**
Changed some code to use up-to-date JavaScript and Ethers, as `var` is widely considered to be obsolete, and Ether's was giving warnings in console since the code was using outdated functions. I also added an optional step to add some styles to the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,55 +198,53 @@ Back in your local text editor in `index.html`, add the following code to your h
...
 
-3. Now, define an ethers provider. In our case it is Ropsten:
+3. Request access to the user's wallet
 
 ```javascript
-var provider = new ethers.providers.Web3Provider(
-  web3.currentProvider,
-  "ropsten"
-);
+(async () => await provider.send("eth_requestAccounts", []))();
```
Comment (haardikk21) [inline README.md]: Do you mind changing this to use `.then()` instead of anonymous async-await so as to make it easier for beginners to understand.

**Review by haardikk21 — CHANGES_REQUESTED** [review]
Thank you, this looks great. Just one comment

File: README.md (hunk 1 above)
Comment (jordszbordsz) [reply README.md]: I can do that but I could also just change the script tag to be `<script type="module">` so top-level await is able to be used, which would make it easier to understand, while also being more up-to-date JavaScript. I'm not good at deciding which is better for newer developers, what do you recommend?

Comment (jordszbordsz) [reply README.md]: @haardikk21 check my latest commits, I made it use `.then` :)

Comment (haardikk21) [reply README.md]: i prefer async/await, but my sole concern here is with the anonymous function. having to do `(async () => await ...)()` is weird unless you understand it, thats why i suggested `.then`

thank you!

**Review by haardikk21 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (haardikk21) [reply README.md]: please let me know your Discord username so I can give you your role

Comment (jordszbordsz) [reply README.md]: jords#0763

**Pull request merged**